src/studying.md

```diff
@@ -253,7 +241,21 @@ contains. Thus, if you select your “French” deck, the subdecks
 “French::Vocab” and “French::My Textbook::Lesson 1” will be shown as
 well.
 
-For new cards and reviews, Anki fetches cards from the decks in
+The way Anki Anki fetches cards from the decks depends on the algorithm used:
+
+- With the v1 scheduler, when a deck has subdecks, the cards will appear from [each deck in
+turn](studying.md#display-order).
+
+- With the [v2 scheduler](https://faqs.ankiweb.net/the-anki-2.1-scheduler.html),
+when a deck has subdecks, reviews are taken from all children decks
+at once. The review limit of the child decks is ignored - only the limit of the
+deck you clicked on applies.
+
+- With the [v3 scheduler](https://faqs.ankiweb.net/the-2021-scheduler.html)
+each child deck's limit is also enforced, and you do not need to see the cards 
+in deck order either. See the [deck options](deck-options.md#review-sort-order) section of the manual for more information.
+
+By default, for new cards and reviews, Anki fetches cards from the decks in
 alphabetical order. So in the above example, you would get cards first
```

Comment:
This line hasn't been true since v1, so perhaps should only mention new cards?

Reply:
Done, thanks!